src/messages/index.ts

```diff
@@ -7,7 +7,7 @@ import { strip } from '../prompt/util/clear.js';
 
 type Message = string | Promise<string>;
 
-export const say = async (msg: Message | Message[] = [], { clear = false, hat = '', stdin = process.stdin, stdout = process.stdout } = {}) => {
+export const say = async (msg: Message | Message[] = [], { clear = false, clothes = { hat: '', tie: '' }, stdin = process.stdin, stdout = process.stdout } = {}) => {
```

Comment:
Even though this is an internal tool, it's best not to break the existing API unless we need to. Keeping `hat` and `tie` separate is a backwards-compatible change.

```suggestion
export const say = async (msg: Message | Message[] = [], { clear = false, hat = '', tie = '', stdin = process.stdin, stdout = process.stdout } = {}) => {
```